public/styles.css

```diff
@@ -0,0 +1,47 @@
+body {
+    font-family: 'Open Sans', sans-serif;
+}
+
+.img-cont {
+  position: absolute;
+  background-image: url('resources/background.jpg');
+  width: 100%;
+  height: 100%;
+  background-size: cover;
+
+  filter: blur(3px);
+}
+
+.welcome-jumbo {
+  position: fixed;
+  width: 100%;
+  opacity: 0.9;
+  top: 15%;
+  text-align: center;
+}
+
+.welcome-owl {
+  width: 125;
+  height: 149;
+}
+
+.welcome-buttons {
+  transform: translateX(38%);
```

Comment:
This seems too hardcoded. Consider the case where I would like to add/remove button. I would have to adjust the position here - could be made a bit more generic